kong-cjson-2.1.0.6-1.rockspec

```diff
@@ -37,11 +38,11 @@ build = {
     },
     install = {
         lua = {
-            ["cjson.util"] = "lua/cjson/util.lua"
+            ["kong.cjson.util"] = "lua/cjson/util.lua"
```

Comment:
@thibaultcha @hishamhm This is an interesting namespace issue. Since the application is in 'kong' now when we create this `kong.cjson` it is inserted in the application namespace, whereas it was named after the organization (I assume).

Opinions on solving/not bothering?